CHANGELOG.md

```diff
@@ -30,7 +30,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-10021: RHCOS node support is dropped until major improvements are made in ROX-8944.
   - The UI shows the node scanning notes in the same manner as image scanning notes.
 - ROX-10097: Updated the base for the docs image from `nginx-118:1-46` to `nginx-120:latest`.
-
+- ROX-10270: In future releases, we plan to deprecate the `RenamePolicyCategory` and `DeletePolicyCategory` methods in the
+`v1/policycategories` endpoint.
+  For questions about this change, please contact the Red Hat support team at support@redhat.com.
```

Comment:
There are examples above (for instance, ll. 10–11) where additional related statements about a change are done with indented "bullets"

```suggestion
   - For questions about this change, please contact the Red Hat support team at support@redhat.com.
```